test/suite/test_chunkcache01.py

```diff
@@ -58,11 +57,11 @@ def get_stat(self, stat):
     def conn_config(self):
         if not os.path.exists('bucket1'):
             os.mkdir('bucket1')
-
         if self.chunk_cache_type == 'dram':
             chunk_cache_extra_config = 'type=DRAM'
         else:
-            chunk_cache_extra_config = 'type=FILE,storage_path=/tmp/chunk_cache_{}'.format(randrange(0, 1000000000))
+            current_dir = os.getcwd()
+            chunk_cache_extra_config = 'type=FILE,storage_path=' +  current_dir + '/WiredTigerChunkCache'
```

Comment:
I don't think we need the `getcwd` - if you just hardcode `WiredTigerChunkCache`, I think it should end up in WT's home directory anyway?

Reply:
No, that wouldn't work as we require an absolute path for `storage_path`.

However, now I am thinking if we should get rid of this config option and simply hardcode `WiredTigerChunkCache` internally in WT.

Reply:
Oh of course - sorry, I forgot about that restriction.

I can see it being acceptable to hardcode a name for the file (maybe in another ticket though), but it's definitely not OK to make the file permanently reside in the DB directory - it's a completely valid use case to have this on a different partition to WT's home directory, since it's potentially very large.

Reply:
Okay, so after reconsidering - 

- If the `storage path` is not passed (i.e. by default), we create WTChunkCache in DB home directory.
- Otherwise, we check whether the path provided is absolute or relative and decide based on that. 

How does that sound?

Reply:
In the above sound good, I can make that change in the format PR.